src/components/Dashboard/RecentPastScheduledObs.vue

```diff
@@ -0,0 +1,68 @@
+<script setup>
+import { useRouter } from 'vue-router'
+import { useObsPortalDataStore } from '../../stores/obsPortalData.js'
+import { useConfigurationStore } from '../../stores/configuration.js'
+import { onMounted, ref } from 'vue'
+import { fetchApiCall } from '../../utils/api.js'
+import ObservationList from './ObservationList.vue'
+
+const router = useRouter()
+const obsPortalDataStore = useObsPortalDataStore()
+const completedObservations = obsPortalDataStore.completedObservations
+const configurationStore = useConfigurationStore()
+const recentRTI = ref([])
+const recentCompleted = ref([])
+
+// Filter out scheduled observations that are older than two weeks
+const twoWeeksAgo = new Date() - 14 * 24 * 60 * 60 * 1000
+const recentObservations = Object.values(completedObservations).filter(observation => new Date(observation.end) > twoWeeksAgo)
+const recentCompletedObservations = recentObservations.filter(observation => observation.state === 'COMPLETED')
+const recentRealTimeObservations = recentObservations.filter(observation => observation.state === 'PENDING')
```

Comment:
Yeah, the variable name makes it seem like you only care about real time or not, but you are filtering on pending or not. Also, it is possible that something persists in pending but is in the past, and will no longer every change (but will probably be deleted after 2 weeks), so you may want to filter on time as well depending on what you are trying to show.

Reply:
@jnation3406  I've added the observation type filter. I'm having trouble understanding what you mean by something persisting but not changing and filtering on time, cause I'm filtering on time (check `recentObservations`) is that what you mean? 🤔

Reply:
What I mean is that its possible to have it PENDING, but have its end time be in the past. If that is the case, there is probably no reason to show it anymore since it will never happen, and will likely get deleted after 2 weeks. There is a small chance due to connectivity issues that the site will come back and switch it to COMPLETED or something, but that is super rare, and if that happens it would not be PENDING anymore. So it should be safe to remove things that are PENDING in the past.